src/front-end/src/components/MentorForm.js

```diff
@@ -103,54 +109,33 @@ class MentorForm extends Component {
                 },
               ]}
             >
-              <Select placeholder="Select timezone.." showSearch>
-                {timeZones.map((zone) => {
-                  return (
-                    <Option key={zone.id} value={getCamelCase(zone.id)}>
-                      {[zone.id, ' ', zone.area]}
-                    </Option>
-                  );
-                })}
-              </Select>
+              <TimezonePicker
+                absolute={false}
+                defaultValue=""
+                placeholder="Select timezone..."
+                onChange={this.handleChange}
+                className="time-zone-picker"
+              />
             </Form.Item>
             <Form.Item
               label="Password"
               name="password"
-              tooltip="Password should be at least 8 characters."
               rules={[
                 {
                   required: true,
                   message: 'Please enter your password for your new account',
                 },
               ]}
-            >
-              <Input type="password" placeholder="Password" minlength="8" />
-            </Form.Item>
-            <Form.Item
-              name="confirmPassword"
-              label="Confirm Password"
-              dependencies={['password']}
-              rules={[
-                {
-                  required: true,
-                  message: 'Please confirm your password',
-                },
-                ({ getFieldValue }) => ({
-                  validator(rule, value) {
-                    if (!value || getFieldValue('password') === value) {
-                      return Promise.resolve();
-                    }
-                    return Promise.reject(
-                      new Error('The password fields do not match'),
-                    );
-                  },
-                }),
-              ]}
```

Comment:
You probably deleted this when fixing the merge conflicts. When you fix the conflicts, you need to make sure you keep both your functionality and the functionality of the branch with which you have a conflict. Here, if we merge this, we are deleting the confirm password functionality, which is present in develop, and should therefore be kept. Please try again to resolve the merge conflict in a way which adds your desired functionality without removing current functionality we have. 

You can watch [this video](https://www.youtube.com/watch?v=JtIX3HJKwfo) about resolving merge conflicts.

Reply:
@wrc9817 If you're not able to resolve JP's comment, let me know and I can fix the form tomorrow morning (well, morning for me, evening for you) :)

Reply:
@jpabadir Thanks for the feedback. I did not realize that.
@KatrinaMart I think I'm good to deal with it. Thank you tho!

Reply:
@wrc9817 I can see that the confirm password field has been added again, but the changes that I had pushed to develop as part of PR #301 have been removed (likely to have been accidentally deleted while trying to fix the merge conflict(s)). If you run the program, you can see that compared to the MenteeForm (which has the changes from PR #301), the MentorForm looks drastically different (for ex. the imported TimeZone library should no longer be on the MentorForm). 
Please let me know if this makes sense, and if you'll be able to incorporate the work that you've done without overriding what was merged in develop from PR #301